app/src/NativeBase/Theme/StaTheme.tsx

```diff
@@ -45,8 +45,11 @@ const StaTheme = extendTheme({
     bgDarkMode: {
       100: common.colours.darkGrey100,
     },
+    inputBorder: {
+      100: '#604696', //dark purple
```

Comment:
Please could you add any new colours here to the `common` variable above -- you can name the colours there, then these comments aren't needed